test/e2e/cron/cron_test.go

```diff
@@ -27,21 +27,47 @@ func (s *CronSuite) TestBasic() {
 		CreateCronWorkflow().
 		Wait(1 * time.Minute).
 		Then().
-		ExpectCron(func(t *testing.T, cronWf *wfv1.CronWorkflowStatus) {
-			assert.True(t, cronWf.LastScheduledTime.Time.After(time.Now().Add(-1*time.Minute)))
+		ExpectCron(func(t *testing.T, cronWf *wfv1.CronWorkflow) {
+			assert.True(t, cronWf.Status.LastScheduledTime.Time.After(time.Now().Add(-1*time.Minute)))
 		})
 }
 
+func (s *CronSuite) TestSuspend() {
+	s.Given().
+		CronWorkflow("@testdata/basic.yaml").
+		When().
+		CreateCronWorkflow().
+		Then().
+		RunCli([]string{"cron", "suspend", "test-cron-wf-basic"}, func(t *testing.T, output string) {
+			assert.Equal(t, "CronWorkflow 'test-cron-wf-basic' suspended", output)
+		}).ExpectCron(func(t *testing.T, cronWf *wfv1.CronWorkflow) {
+			assert.True(t, cronWf.Spec.Suspend)
+		})
+}
+
+func (s *CronSuite) TestResume() {
+	s.Given().
+		CronWorkflow("@testdata/basic.yaml").
+		When().
+		CreateCronWorkflow().
+		Then().
+		RunCli([]string{"cron", "resume", "test-cron-wf-basic"}, func(t *testing.T, output string) {
```

Comment:
Interesting... perhaps we can refactor the `cli_test.go`?

Reply:
Sure, however that is still only on the `apiserverimpl` branch. Maybe we can merge this, then I'll make the changes on that branch?

Reply:
ah - of course - we can merge from `master` into `apiserverimpl`